10/umbraco-cms/extending/language-files.md

```diff
@@ -150,7 +150,7 @@ localizationService.localize("dialog_myKey").then(function(value){
 
 ## Help keep the language files up to date
 
-As Umbraco is a continually evolving product it is inevitable that new text is added on a fairly regular basis to the English language version of these files. This may mean that some of the above languages are no longer up to date.
+As Umbraco is a continually evolving product, it is inevitable that new text is added. This happens on a regular basis to the English language version of these files. This may mean that some of the above languages are no longer up to date.
```

Comment:
the sentence has 2 "this" one after another. I took the text from the updated v14 article: https://docs.umbraco.com/umbraco-cms/v/14.latest-rc/extending-backoffice/localization/net-localization#help-keep-the-language-files-up-to-date
```suggestion
As Umbraco is a continually evolving product it is inevitable that new text is added regularly to the English language version of these files. This may mean that some of the above languages are no longer up to date.
```